### ISSUE TITLE
I have an issue with this plugin, because it simply wont work with every option i tried !

### ISSUE THREAD

**Issue by AlenIdealen** [Gix075/jqueryIntroLoader#11]
**For example i want to test it on my Login/Register index.php page :**

It just wont work and i have checked online documentation...i have tried git hub solutions...i have tried copying direct demo files to use with my page...nothing works...also tried. This is by far most time consuming simple thing i couldn't get to work at all...

<div id="element" class="introLoading"></div>
Javascript

$(document).on('ready' , function() {
    $("#element").introLoader();
});

<?php 
	/* Main page with two forms: sign up and log in */
	require 'db.php';
	session_start();
?>

<!DOCTYPE html>
<html>

<head>

  <title>Prijava / Registracija</title>

        
        <link href="preloader/css/introLoader.min.css" rel="stylesheet">
        
        <script src="preloader/jquery.introLoader.pack.min.js"></script>
  
        
  
  <link rel="stylesheet" href="css/animirantext.css">
  
  <?php include 'css/css.html'; ?>
</head>

<?php 
if ($_SERVER['REQUEST_METHOD'] == 'POST') 
{
    if (isset($_POST['login'])) 
	{	//user logging in
        require 'login.php';
    }
    
    elseif (isset($_POST['register'])) 
	{	//user registering
        require 'register.php';
    }
}
?>

<body>
        
    
        
        
        <div id="element"></div>
        
        <script>
            
                $(document).ready( function() {
                        $("#element").introLoader({
                                    
                    animation: {
                        name: 'doubleLoader',
                        options: {
                            onBefore: function() {$("#element").addClass('introLoading');}
                        }
                    },
                            exitFx:'fadeOut',
                            ease: "easeInOutCirc",
                            style: 'ocean zebra',
                            delayBefore: 500,
                            exitTime: 300,
                            progbarTime: 700,
                            progbarDelayAfter: 400,
                            preventScroll: true
                        });
                    
                });
			
		</script>

        

        
	
	<div class="form">
	
		
		<ul class="tab-group">
			<li class="tab"><a href="#signup">Registracija</a></li>
			<li class="tab active"><a href="#login">Prijava</a></li>
		</ul>
		
		
		<div class="tab-content">
		
			
			<div id="login">   
                                                                <h1>Dobrodošli nazaj!</h1>
                                                                <form><br>
                                                                <div class="content">
                                                                <div class="content__container">
                                                                    <p class="content__container__text">
                                                                        Pozdravljeni
                                                                    </p>

                                                                  <ul class="content__container__list">
                                                                    <li class="content__container__list__item">Popotniki !</li>
                                                                    <li class="content__container__list__item">Alpinisti !</li>
                                                                    <li class="content__container__list__item">Pohodniki !</li>
                                                                    <li class="content__container__list__item">Izletniki !</li>
                                                                  </ul>
                                                                </div>
                                                              </div>   
                                                        </form>
                                                                <form action="index.php" method="post" autocomplete="off"><br><br>
					<div class="field-wrap">
						<label>E-poštni naslov<span class="req">*</span></label>
						<input type="email" required autocomplete="off" name="email"/>
					</div>
					
					<div class="field-wrap">
						<label>Geslo<span class="req">*</span></label>
						<input type="password" required autocomplete="off" name="geslo"/>
					</div>
				
					<p class="forgot"><a href="forgot.php">Ste pozabili geslo?</a></p>
					
					<button class="button button-block" name="login" />Prijava</button>
				</form>
			</div>

			
			
			<div id="signup">   
				<h1>Registracija</h1>
                                
                                
				<form><br>
                                                                <div class="content">
                                                                <div class="content__container">
                                                                  <p class="content__container__text">
                                                                    Pozdravljeni
                                                                  </p>

                                                                  <ul class="content__container__list">
                                                                    <li class="content__container__list__item">Popotniki !</li>
                                                                    <li class="content__container__list__item">Alpinisti !</li>
                                                                    <li class="content__container__list__item">Pohodniki !</li>
                                                                    <li class="content__container__list__item">Izletniki !</li>
                                                                  </ul>
                                                                </div>
                                                              </div>   
                                                        </form>
                                
                                
                                
                                                                        <form action="index.php" method="post" autocomplete="off"><br><br>
					<div class="top-row">
						<div class="field-wrap">
							<label>Ime<span class="req">*</span></label>
							<input type="text" required autocomplete="off" name='ime' />
						</div>
						
						<div class="field-wrap">
							<label>Priimek<span class="req">*</span></label>
							<input type="text"required autocomplete="off" name='priimek' />
						</div>
					</div>
					
					<div class="field-wrap">
						<label>E-poštni naslov<span class="req">*</span></label>
						<input type="email"required autocomplete="off" name='email' />
					</div>

					<div class="field-wrap">
						<label>Nastavi geslo<span class="req">*</span></label>
						<input type="password"required autocomplete="off" name='geslo'/>
					</div>

					<button type="submit" class="button button-block" name="register" />Registracija</button>
				</form>
			</div>
		</div> 
		
		
		<br> <hr> <br>	
		<a href="popotnik.php"><button class="button button-block" name="portal"/>Portal</button></a>
		
	</div> 

	
	<script src='http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js'></script>
    <script src="js/index.js"></script>
    
  
</body>

</html>

**Comment by Gix075** [comment]
Hi, try to load `<script src="preloader/jquery.introLoader.pack.min.js"></script>` after jQuery.

**Comment by AlenIdealen** [comment]
Yeah i managed to figure out something with my other custom preloaders, so i think yours should work aswell...will def be back to test your amazing preloader, because i have to include it :) !

**Comment by AlenIdealen** [comment]
**IT worked now perfectly...just came back home and tested this on my email.php**


<?php
	$msg = "";

	if (isset($_POST['submit'])) {

		require 'phpmailer/PHPMailerAutoload.php';

		function sendemail($to, $from, $fromName, $body, $attachment = "") {
			$mail = new PHPMailer();
			$mail->setFrom($from, $fromName);
			$mail->addAddress($to);
			$mail->addAttachment($attachment);
			$mail->Subject = 'Kontaktni E-mail obrazec @Slovenski Popotnik';
			$mail->Body = $body;
			$mail->isHTML(false);

			return $mail->send();
		}

		$name = $_POST['username'];
		$email = $_POST['email'];
		$body = $_POST['body'];

		$file = "attachment/" . basename($_FILES['attachment']['name']);
		if (move_uploaded_file($_FILES['attachment']['tmp_name'], $file)) {
		    if (sendemail('slovenski.pohodnik@gmail.com', $email, $name, $body, $file)) {
				$msg = 'Email sent!';
				sendemail('slovenski.pohodnik@gmail.com', 'SlovenskiPopotnik@gmail.com', 'SlovenskiPopotnik', 'Pozdrav, prejeli smo nov E-mail');
			} else {
                                                            $msg = 'Email failed!';}
                } else {
			$msg = "Please check your attachment!";
	}
        }
?>

<!doctype html>
<html>
<head>
    
    
                
        
        <link href="preloader/introLoader.min.css" rel="stylesheet">
        
       
                

    
<title>E-pošta @Slovenski Popotnik</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="keywords" content="Ravishing Contact Form Responsive, Login form web template,Flat Pricing tables,Flat Drop downs  Sign up Web Templates, Flat Web Templates, Login signup Responsive web template, Smartphone Compatible web template, free webdesigns for Nokia, Samsung, LG, SonyEricsson, Motorola web design" />
<script type="application/x-javascript"> addEventListener("load", function() { setTimeout(hideURLbar, 0); }, false); function hideURLbar(){ window.scrollTo(0,1); } </script>
<script src="popotnik/js/jquery-1.11.1.min.js"></script>
<script data-require="jquery@2.2.0" data-semver="2.2.0" src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>

<link href="//fonts.googleapis.com/css?family=Raleway:100,200,300,400,500,600,700,800,900" rel="stylesheet">
<link href="//fonts.googleapis.com/css?family=Roboto+Condensed:300,400,700" rel="stylesheet">


<link href="css/font-awesome.min.css" rel="stylesheet" type="text/css" media="all" />
<link href="css/style_email.css" rel="stylesheet" type='text/css' media="all" />



</head>
<body>
    
<div id="element" class="introLoading"></div>  
    
<h1 class="w3layouts" style="color:mintcream; font-family:Imprint MT Shadow">Kontaktni E-mail obrazec</h1>
<div class="content-w3ls">
	<div class="content1-agile">
		<div class="info-w3l">
			<h2 class="wthree">Pošljite sporočilo</h2>
                                                <form action="email.php" method="post" class="form-agileits" enctype="multipart/form-data">
				<input type="text" id="username" name="username" placeholder="Uporabniško ime" title="Prosimo vnesite samo Uporabniško ime" required="">
				<input type="email" id="email" name="email" placeholder="E-pošta@primer.com" title="Prosimo vnesite veljavni E-poštni naslov" required="">
                                                                <textarea type="body" id="message" name="body" placeholder="Vaše sporočilo" title="Prosimo napišite kaj nam želite sporočiti ali kaj vas zanima"required=""></textarea>
                                
                                                        
                                
                                                                <label for="file" class="input-label">
                                                                <i class="fa fa-upload" aria-hidden="true"></i>
                                                                <span id="label_span">Izberite datoteko / sliko</span>
                                                                </label>
                                                                <input type="file" name="attachment" id="file" multiple="attachment" required="">
                                                                <div id="selectedFiles"></div>
                                                         
                                                       
                                                       
                                                       
                                                                <div style="line-height:130%;"><br></div>
				<input type="submit" name="submit" class="sign-in" value="Pošlji na @SlovenskiPopotnik">
                                                             
                                                          
                                                </form>
                        
                                                
                                                <div style="line-height:50%;">
                                                        <br>
                                                </div>
                                                
                                                        <form action="popotnik.php" method="post" class="form-agileits">
                                                                <input action="popotnik.php" type="submit" class="sign-in" value="Slovenski Popotnik">
                                                        </form>
                        
			<p class="agileinfo">Ali</p>
			<ul class="social-w3-agileits">
				<li><a href="https://www.facebook.com/SlovenskiPopotnik/"><i class="fa fa-facebook"></i></a></li>
				<li><a href="#"><i class="fa fa-twitter"></i></a></li>		
				<li><a href="#"><i class="fa fa-google-plus"></i></a></li>
				<li><a href="#"><i class="fa fa-linkedin"></i></a></li>
			</ul>
		</div>	
	</div>
	<div class="content2-agile">
		
	</div>
	<div class="clear"></div>
</div>
<p class="copyright-agileits">© 2017 Slovenski Popotnik. Vse pravice pridržane | Design by <a href="https://w3layouts.com/" target="_blank">w3layouts</a></p>


                    <script src="js/email.js"></script> 
                    
        <?php echo $msg; ?>

                    <script src="http://ajax.googleapis.com/ajax/libs/jquery/2.0.0/jquery.min.js"></script>
                    <script src="preloader/jquery.introLoader.pack.min.js"></script>
                    
                    <script type="text/javascript">   $("#element").introLoader({
    animation: {
        name: 'doubleLoader',
        options: {
            exitFx:'fadeOut',
            ease: "easeInOutCirc",
            style: 'ocean zebra',
            delayBefore: 500,
            exitTime: 300,
            progbarTime: 700,
            progbarDelayAfter: 400,
            preventScroll: true
        }
    }
});

</script>
</body>


</html>
